release-notes.md

```diff
@@ -1,3 +1,14 @@
+# Rigel-18.16.0
```

Comment:
move to the new version section before merge
also there are a lot missing release notes here for previous features others developed for 18.16, make sure nothing is missing